plugins.xml

```diff
@@ -9768,6 +9768,41 @@
 			<description>Minor improvements for Portuguese translations.</description>
 		</release>
 	</plugin>
+	<plugin category="generic" product="webOfScience">
```

Comment:
Hi @Alex-nz, thanks a lot for the contribution!
I went through the plugin code and would have a few comments/questions:
- The handler https://github.com/clarivate/wos_reviewer_recognition_plugin_ojs_3/blob/main/WOSHandler.php is missing the authorization function. This way everybody could call the function `exportReview`. I believe that `addRoleAssigment` should be used, so that only reviewers (an user with a reviewer role) can call the function, and it should also be checked if the request context corresponds to the submission context (having the submission from the reviewAssignment) -- so that a reviewer from one journal cannot call that function for a review in another journal (that maybe has no plugin enabled) -- e.g. if the parameter `reviewId`, that the function expects, would be manipulated. I believe the WOS service would surely not work correctly for example in case the user is not a reviewer any more, or for a review in another journal, but the safety should be ensured by coding it.
- When are these lines the case: https://github.com/clarivate/wos_reviewer_recognition_plugin_ojs_3/blob/main/WOSHandler.php#L191 and https://github.com/clarivate/wos_reviewer_recognition_plugin_ojs_3/blob/main/WOSHandler.php#L214 -- when and how would `$_SERVER["HTTP_WOS_URL"]` be set? 
- Instead of using php curl you should rather use Guzzle -- you can get the HTTP client from the `Application` class. This client is properly configured for proxies if the server is behind one.
- I think `<certification type="reviewed"/>` should be used below.
Please let me know if you would have any questions and what do you think about the suggestions...
Thanks a lot!

Reply:
Thanks for the review @bozana. I would have to defer answering your comments to the developer who worked on this plugin. @v0k1c when you are back from vacation, can you please look into this?